pulsar-client/src/main/java/org/apache/pulsar/client/impl/BinaryProtoLookupService.java

```diff
@@ -131,7 +132,7 @@ public BinaryProtoLookupService(PulsarClientImpl client, String serviceUrl, bool
             }).exceptionally((sendException) -> {
                 // lookup failed
                 log.warn("[{}] failed to send lookup request : {}", destination.toString(), sendException.getMessage(),
```

Comment:
better to omitting exception in the log message, rather than using `null`. because `null` is a bit confusing for users.

```
if (sendException instanceof ClosedChannelException) {
    log.warn("[{}] failed to send lookup request.", destination.toString(), sendException.getMessage());
}

Reply:
the `null` is not getting actually printed, the exception message is always printed (and that's what gets added in the `{}`). This is just to prevent the stack trace after the log line